utilities/src/with_std/metrics.rs

```diff
@@ -455,50 +547,50 @@ build_counter_vec!(
 );
 build_counter_vec_struct!(
 	CEREMONY_PROCESSED_MSG,
-	CeremonyProcessedMsgDrop,
+	CeremonyProcessedMsg,
 	"ceremony_msg",
 	"Count all the processed messages for a given ceremony",
-	true,
-	["chain", "ceremony_id", "ceremony_type"]
+	false,
+	["chain", "ceremony_type"]
 );
 build_counter_vec_struct!(
 	CEREMONY_BAD_MSG,
-	CeremonyBadMsgNotDrop,
+	CeremonyBadMsg,
 	"ceremony_bad_msg",
 	"Count all the bad msgs processed during a ceremony",
 	false,
 	["chain", "reason"],
 	["chain"] //const labels
 );
-build_gauge_vec_struct!(
+build_histogram_vec_struct!(
 	CEREMONY_DURATION,
-	CeremonyDurationDrop,
+	CeremonyDuration,
 	"ceremony_duration",
 	"Measure the duration of a ceremony in ms",
-	true,
-	["chain", "ceremony_id", "ceremony_type"]
+	["chain", "ceremony_type"],
+	(vec![1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0])
 );
 build_gauge_vec_struct!(
 	CEREMONY_TIMEOUT_MISSING_MSG,
-	CeremonyTimeoutMissingMsgDrop,
+	CeremonyTimeoutMissingMsg,
 	"ceremony_timeout_missing_msg",
 	"Measure the number of missing messages when reaching timeout",
-	true,
-	["chain", "ceremony_id", "ceremony_type", "stage"],
-	["chain", "ceremony_id", "ceremony_type"]
+	false,
+	["chain", "ceremony_type", "stage"],
+	["chain", "ceremony_type"]
 );
-build_gauge_vec_struct!(
+build_histogram_vec_struct!(
 	STAGE_DURATION,
-	StageDurationDrop,
+	StageDuration,
 	"stage_duration",
 	"Measure the duration of a stage in ms",
-	true,
-	["chain", "ceremony_id", "stage", "phase"], //phase can be either receiving or processing
-	["chain", "ceremony_id"]
+	["chain", "stage", "phase"], //phase can be either receiving or processing
+	["chain"],
+	(vec![1.0, 2.0, 3.0])
```

Comment:
Lets change these to be in seconds, and use 4, 8, 16, 32, 64, 128, 256, 512, 1024, for both Histograms